packages/@aws-cdk/aws-ecs/lib/base/task-definition.ts

```diff
@@ -464,6 +509,43 @@ export interface Host {
   readonly sourcePath?: string;
 }
 
+/**
+ * The port number as well as the protocol of a container host port.
+ *
+ * @internal
+ */
+export interface HostPort {
```

Comment:
Instead of marking as `@internal`, better to just not `export` it (or `export` from a file that is itself not exported in `index.ts`).

Reply:
Done:+1: